src/LoginPage/components/RegisterForm.js

```diff
@@ -21,36 +22,33 @@ import "../css/register.css";
 import createFragment from "react-addons-create-fragment";
 import { ValidatorComponent } from 'react-material-ui-form-validator';
 import holder from "../images/holder.jpg";
+import { fetchMeetingOptions } from "../../actions";
 
+function mapStateToProps ({ allMeetingOptions }){
+    return {allMeetingOptions}
```

Comment:
For readability, I'd recommend using a consistent styling of object literals and spacing. The standard is typically using spacing.
e.g. `{ allMeetingOptions }` versus `{allMeetingOptions}`.

https://eslint.org/docs/rules/object-curly-spacing